services/scan/src/importer.ts

```diff
@@ -136,6 +137,8 @@ export class Importer {
    */
   configure(electionDefinition: ElectionDefinition): void {
     this.workspace.store.setElection(electionDefinition);
+    // Central scanner only uses all precinct mode, set on every configure
+    this.workspace.store.setPrecinctSelection(ALL_PRECINCTS_SELECTION);
```

Comment:
Since this added logic is specific to the central scanner, we may want to move it up to `services/scan/src/central_scanner_app.ts` instead of keeping it in this lower, machine-agnostic layer

Reply:
I originally had it in the higher layer but it had to be in a few places - both on initial setup and any reconfigures - to work properly.

The main point to make is that, I believe, `importer.ts` is not actually machine-agnostic. It is only used for VxCentralScan. I think @jonahkagan told me this. Will let him or @eventualbuddha confirm.

Reply:
I was confused by this at first and if `importer.ts` is machine specific now, we might rename the file to reflect that.

Reply:
Ah interesting - I like the idea of renaming then! Maybe `central_scanner_importer.ts` / `CentralScanImporter`

Reply:
https://github.com/votingworks/vxsuite/issues/2532

Reply:
Yep, importer is now only used by the central scanner. Renaming it makes sense!